sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/AdministrationModelFactory.cs

```diff
@@ -7,11 +7,12 @@
 
 using System.Collections.Generic;
 using System.Linq;
+using Azure.Security.KeyVault.Administration;
 
-namespace Azure.Security.KeyVault.Administration
+namespace Azure.Security.KeyVault.Administration.Models
```

Comment:
Previously it is a mistake because this SDK has enabled `model-namespace: true`, but the namespace of the model factory is generated without `.Models`.
But this is a GA package, we need to change it back using customization code.